addons/account_followup/account_followup.py

```diff
@@ -77,432 +57,152 @@ def _get_default_template(self, cr, uid, ids, context=None):
-                                         "Can be practical to set manually e.g. to see if he keeps "
-                                         "his promises."),
-        'unreconciled_aml_ids':fields.one2many('account.move.line', 'partner_id', domain=['&', ('reconciled', '=', False), '&', 
-                            ('account_id.deprecated','=', False), '&', ('account_id.internal_type', '=', 'receivable')]), 
-        'latest_followup_date':fields.function(_get_latest, method=True, type='date', string="Latest Follow-up Date", 
-                            help="Latest date that the follow-up level of the partner was changed", 
-                            store=False, multi="latest"), 
-        'latest_followup_level_id':fields.function(_get_latest, method=True, 
-            type='many2one', relation='account_followup.followup.line', string="Latest Follow-up Level", 
-            help="The maximum follow-up level", 
-            store={
-                'res.partner': (lambda self, cr, uid, ids, c: ids,[],10),
-                'account.move.line': (_get_partners, ['reconciled', 'followup_line_id'], 10),
-                'account.partial.reconcile': (_get_partners_from_partial_rec, [], 10),
-            }, 
-            multi="latest"), 
-        'latest_followup_level_id_without_lit':fields.function(_get_latest, method=True, 
-            type='many2one', relation='account_followup.followup.line', string="Latest Follow-up Level without litigation", 
-            help="The maximum follow-up level without taking into account the account move lines with litigation", 
-            store={
-                'res.partner': (lambda self, cr, uid, ids, c: ids,[],10),
-                'account.move.line': (_get_partners, ['reconciled', 'followup_line_id'], 10),
-            }, 
-            multi="latest"),
-        'payment_amount_due':fields.function(_get_amounts_and_date, 
-                                                 type='float', string="Amount Due",
-                                                 store = False, multi="followup", 
-                                                 fnct_search=_payment_due_search),
-        'payment_amount_overdue':fields.function(_get_amounts_and_date,
-                                                 type='float', string="Amount Overdue",
-                                                 store = False, multi="followup", 
-                                                 fnct_search = _payment_overdue_search),
-        'payment_earliest_due_date':fields.function(_get_amounts_and_date,
-                                                    type='date',
-                                                    string = "Worst Due Date",
-                                                    multi="followup",
-                                                    fnct_search=_payment_earliest_date_search),
-        }
-
-
-class account_config_settings(osv.TransientModel):
+
+    def get_partners_in_need_of_action(self):
```

Comment:
too weird having that function overwritten and its signature changed in the same time. Please rename that function in order to make it clear it doesn't do the same.

also, instead of the sql query you should be able to use get_followup_lines_domain()